### PR TITLE
Document latest `DropdownMenu`/`ContextMenu` changes (`CheckboxItem` indeterminate)

### PR DESCRIPTION
**Pull request by benoitgrelard**
https://github.com/radix-ui/primitives/pull/1624

Changelog entry: https://github.com/radix-ui/website/pull/493/commits/b6669c8143d383c814a67b58f727194e806214d8
Docs changes: https://github.com/radix-ui/website/pull/493/commits/9969dd63445ef73627341fe5671879b05d45c249

### CODE REVIEW

File: data/primitives/components/context-menu/2.0.0.mdx [hunk 1]
```diff
@@ -0,0 +1,1353 @@
...
+
+```jsx hero template=ContextMenu
+
+```
+
+<Highlights
+  features={[
+    'Supports submenus with configurable reading direction.',
+    'Supports items, labels, groups of items.',
+    'Supports checkable items (single or multiple), can be indeterminate too.',
```
Comment (andy-hook) [inline data/primitives/components/context-menu/2.0.0.mdx:27]: I wonder if we even need to mention indeterminate here or if it's already implied.

**Review by andy-hook — APPROVED** [review]

File: data/primitives/components/context-menu/2.0.0.mdx (hunk 1 above)
Comment (benoitgrelard) [reply data/primitives/components/context-menu/2.0.0.mdx:27]: I added because we have it on Checkbox

Comment (benoitgrelard) [reply data/primitives/components/context-menu/2.0.0.mdx:27]: Let me know.

Comment (andy-hook) [reply data/primitives/components/context-menu/2.0.0.mdx:27]: Ah that's fair enough, in that case how'd you feel about?

`Supports checkable items (single or multiple) with optional indeterminate state.`

Comment (benoitgrelard) [reply data/primitives/components/context-menu/2.0.0.mdx]: Done!

**Pull request merged**